data/BalatonFelvidek.md

```diff
@@ -1 +1,11 @@
-Ez egy lehetőség a Balaton-felvidéki utak alaposabb bejárására. Az egyes szakaszokról a későbbiekben tervezünk bővebben írni.
\ No newline at end of file
+Ez a bejáróút egy sokkal érdekesebb, izgalmasabb alternatíva Veszprém és Tapolca összekötésére a [77-es útnál](#77). Az útvonal mentén végig változatos, gyönyörű kilátásokban lehet részünk, miközben kanyargunk, vagy bukkanókat veszünk. A bejáróút szakaszai a tempós, erdei [73-as út](#73), a [Füred és Tótvázsony közti](#BalatonfuredTotvazsony) szőlők közti kanyargós és a [Tótvázsony és Dörgicse](#TotvazsonyDorgicse) illetve [Köveskál és Gyulakeszi](#KoveskalGyulakeszi) közti bukkanós-kanyargós, gyönyörű kilátású szakaszok.
+
+A bejáróút felfúzhető körtúrába a [77-es úttal](#77), bár ez kevésbé izgalmas útvonal.

```

Comment:
A bejáróút felfűzhető a - kevésbé izgalmas -  [77-es úttal](#77) is a körtúrába.